app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java

```diff
@@ -149,6 +152,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 saveTags();
                 startImagesActivity();
                 return true;
+            case CLEAR_ALL_TAGS:
+                clearAllTags();
+                return true;
```

Comment:
We should think about moving toolbar (buttons) logic to separate viewModel-like class. This is only a suggestion to think about.